staging/src/k8s.io/apiserver/pkg/audit/request.go

```diff
@@ -171,7 +171,17 @@ func LogResponseObject(ae *auditinternal.Event, obj runtime.Object, gv schema.Gr
 		return
 	}
 	if status, ok := obj.(*metav1.Status); ok {
-		ae.ResponseStatus = status
+		if ae.Level.Less(auditinternal.LevelRequestResponse) {
+			// selectively copy the bounded fields.
+			ae.ResponseStatus = &metav1.Status{
+				Status: status.Status,
+				Reason: status.Reason,
+				Code:   status.Code,
+			}
+		} else {
```

Comment:
I think we should get rid of the else case. This just duplicates the status when it's already being returned in the Response field.

Reply:
> This just duplicates the status when it's already being returned in the Response field.

We have a `return` in line 184, so it's not duplicated.

If we delete it, it will be diffacult for users to query the http response code in response level audit events. WDYT @tallclair